### ISSUE TITLE
Getting all badges will also (visual) quarantine your account, but after that all badges including quarantine will disappear.

### ISSUE THREAD

**Issue by FenixGlobElite** [hxr404/Discord-Console-hacks#59]
## Reproduce

Use console command below:
```
window.webpackChunkdiscord_app.push([[Math.random()], {}, (req) => {for (const m of Object.keys(req.c).map((x) => req.c[x].exports).filter((x) => x)) {if (m.default && m.default.getCurrentUser !== undefined) {return m.default.getCurrentUser().flags = -1;}if (m.getCurrentUser !== undefined) {return m.getCurrentUser().flags = -1}}}]);window.webpackChunkdiscord_app.push([[Math.random()], {}, (req) => {for (const m of Object.keys(req.c).map((x) => req.c[x].exports).filter((x) => x)) {if (m.default && m.default.getCurrentUser !== undefined) {return m.default.getCurrentUser().public_flags += 1;}if (m.getCurrentUser !== undefined) {return m.getCurrentUser().public_flags += 1}}}]);
```

**Comment by Visual9999** [comment]
thats not correct they will stay but client crashes

**Comment by FenixGlobElite** [comment]
> 

For me, client doesn't crash, they just disappear

**Comment by spinfal** [comment]
That's because you are giving yourself every flag, and account quarantine is a flag.

Check out my PR https://github.com/hxr404/Discord-Console-hacks/pull/57/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R157-R164

**Comment by hxr404** [comment]
fixed by #57 / #65
